settings.json

```diff
@@ -0,0 +1,11 @@
+{
+  "TWILIO": {
+    "FROM": "meteor-whatsapp",
+    "SID": "",
+    "TOKEN": ""
+  },
+  "ACCOUNTS_PHONE": {
+    "ADMIN_NUMBERS": ["123456789", "987654321"],
+    "MASTER_CODE": "1234"
+  }
+}
```

Comment:
I can remove this.  It was used when I tested the app without typing a real number.